src/Tizen.Applications.EventManager/Tizen.Applications/ApplicationEventManager.cs

```diff
@@ -0,0 +1,105 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Applications.EventManager
+{
+    /// <summary>
+    /// The EventManager Class provides functions to broadcast user-defined event.
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// public class EventManagerSample
```

Comment:
you need to use CDATA section. 
http://suprem.sec.samsung.net/confluence/display/API/4.+Documentation+Guide#id-4.DocumentationGuide-example%3Cexample%3E%3Ccode%3E...%3C/code%3E%3C/example%3E-CodeExample